Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -421,8 +446,20 @@ class ValetIntegrationTests: XCTestCase
         let removeQueue = DispatchQueue(label: "Remove Object Queue", attributes: .concurrent)
 
         for _ in 1...50 {
-            setQueue.async { XCTAssertTrue(self.valet.set(string: self.passcode, forKey: self.key)) }
-            removeQueue.async { XCTAssertTrue(self.valet.removeObject(forKey: self.key)) }
+            setQueue.async {
+                do {
```

Comment:
we need to use a `do/catch` block because we're within an `async` `() -> Void` block that can't `throw`.